views/explorer.tmpl

```diff
@@ -16,61 +16,48 @@
             {{$Offset := (subtract .BestBlock $topBlock)}}
             {{$pendingBlocks := ((index .Data 0).Height)}}
             <div class="d-flex justify-content-between align-items-end">
-                <span class="h2 d-flex pt-4 pb-1 pr-2">Blocks</span>
-
+                <span class="h2 d-flex pt-4 pb-1 pr-2">
+                    Per blocks
+                    <span class="dcricon-info ml-3 mt-1"></span>
```

Comment:
It's better as just "Blocks", but leave the icon. Maybe make the icon a little bit smaller.